crates/holochain_state/src/source_chain.rs

```diff
@@ -731,7 +739,8 @@ where
                 WHERE
                 Action.author = :author
                 AND
-                (:range_start IS NULL AND :range_end IS NULL AND :range_start_hash IS NULL AND :range_end_hash IS NULL AND :range_prior_count IS NULL)
+                (
```

Comment:
is this brace and its corresponding close inserted to fix a precedence bug?

Reply:
Yes, this is a bug fix.